packages/sdk/src/tests/FuelWallet.test.ts

```diff
@@ -20,10 +20,25 @@ describe('Fuel Wallet SDK multiple connectors', () => {
     window.localStorage.clear();
   });
 
+  test('Create using default connectors', async () => {
+    const fuel = new Fuel();
+    const connectors = await fuel.connectors();
+    expect(connectors.length).toBe(2);
+  });
+
+  test('Create using default connectors devMode', async () => {
+    const fuel = new Fuel({
+      devMode: true,
+    });
+    const connectors = await fuel.connectors();
+    expect(connectors.length).toBe(3);
+  });
```

Comment:
improve a bit checking the dev connector itself
```suggestion
  test('devMode should include Fuel Wallet Development', async () => {
    const fuel = new Fuel({
      devMode: true,
    });
    const connectors = await fuel.connectors();
    expect(connectors.length).toBe(3);
    expect(connectors[3].name).toBe('Fuel Wallet Development');
  });
```

Reply:
we can also add a test using only 1 custom connector instead of the default ones?

Reply:
@LuizAsFight we already have a test for this.